Library/Homebrew/extend/ENV/std.rb

```diff
@@ -303,6 +303,10 @@ def libstdcxx
     end
   end
 
+  def java_cache
+    append "_JAVA_OPTIONS", "-Duser.home=#{HOMEBREW_CACHE}/java_cache"
+  end
```

Comment:
Only there are subtle differences in the implementation of `deparallelize` in `std` and `super`, while this method is identical and thus shouldn't be duplicated. (Not saying that those differences in `deparallelize` should exist. But moving/unifying `deparallelize` is not what this PR is about.)